src/lua/zencode_foreach.lua

```diff
@@ -64,3 +64,8 @@ Foreach("'' in sequence from '' to '' with step ''", function(name, from_name, t
         end
     end
 end)
+
+When("exit from foreach loop", function()
```

Comment:
best use:

'exit foreach loop`

which will support also `exit the`

unsure if to use `break` as a verb, more used in other langs

Reply:
If we are unsure qe can have them as alias both `exit` and `break`

Reply:
Yes to aliasing both `exit` and `break`.

I checked and the word `loop` is not mentioned anywhere so:

```
break foreach
exit foreach
```

One doubt left since `endforeach` is used as one word to mark end of branch... I think there will be confusion later on about having the word separated, as well between `exit` and `end`.